Gemfile

```diff
@@ -2,39 +2,14 @@
 
 source 'https://rubygems.org'
 
-git_source(:github) do |repo_name|
-  repo_name = "#{repo_name}/#{repo_name}" unless repo_name.include?('/')
-  "https://github.com/#{repo_name}.git"
-end
-
-# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
+gem 'coffee-rails', '~> 4.2'
+gem 'haml'
+gem 'haml-rails'
+gem 'mysql2'
```

Comment:
Switching `sqlite3` to `mysql2` is the only real change here. Everything else is cleanup/alphabetization.